tcms/management/management/commands/baseurl.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = ("Sets the base URL of Kiwi TCMS instance. "
+            "If no arguments given returns current base URL."
+            )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'baseurl', nargs='?', default=None,
+            help='Base URL of Kiwi TCMS instance',
+        )
+
+    def handle(self, *args, **kwargs):
+        site = Site.objects.get(id=settings.SITE_ID)
+        if not kwargs['baseurl']:
+            self.stdout.write('URL: "%s", name: "%s"' % (site.domain, site.name))
```

Comment:
let's just return the domain name here without anything else so scripts who want to use this command don't have to parse the output.